src/app/study-sequence/study-sequence.component.scss

```diff
@@ -0,0 +1,43 @@
+.back-to-view{
+    border-style: dashed;
+    height: 44px;
+}
+
+.container{
+    display: grid;
+    //Makes a grid with 3 columns
+    grid-template-columns: auto auto auto;
+    height: calc(100vh - 100px);
+}
+
+.card-pool{
+    outline: 1px solid blue;
+    grid-column-start: 1;
+    grid-column-end: 3;
+    text-align: center;
+}
+
+.sequence-builder{
+    outline: 1px dashed magenta;
+    grid-column-start: 3;
+    grid-column-end: 4;
+    display: grid;
+    grid-template-rows: auto auto auto;
+}
+
+.sequence-select{
```

Comment:
The top of the select sequence should butt up next to the navbar. To accomplish this, you might need to put back to view in the container grid you made.